src/pages/posts/[...slug].astro

```diff
@@ -41,10 +41,11 @@ const jsonLd = {
     url: Astro.site,
   },
   datePublished: formatDateToYYYYMMDD(entry.data.published),
+  inLanguage: entry.data.language.replace('_', '-'),
```

Comment:
My apologies for not being clear. I meant that if the `lang` in the frontmatter is not set, use the default one from `config.ts`.

Reply:
Sorry, I misunderstood your request. 🥲 
It has been fixed in the latest commit